altiProfilAdmin/controllers/config.classic.php

```diff
@@ -52,7 +51,20 @@ function index() {
                 $form->setData( $ctrl->ref, $val );
             }
         }
-
+        $tableName =  $form->getData('altiProfileTable');
+        if ($form->getData('altiProfileProvider') == 'database' && !empty($tableName)) {
+            try{
+              $sql = 'SELECT rast FROM "'.$tableName.'" limit 0';
+              $cnx = \jDb::getConnection( 'altiProfil' );
+              $qResult = $cnx->query( $sql);
+              $form->setData('connection_check', jLocale::get('altiProfilAdmin~admin.form.connection_check.ok'));
```

Comment:
```suggestion
              $form->setData('connection_check', '✓');
```

Reply:
I kept the locale, as the value can be adapted  by any translator without  editing the code.